### PR TITLE
fix 404 for rails5

### PR DESCRIPTION
**Pull request by samsonasu**

### CODE REVIEW

**Comment by boatrite** [comment]
:boat:

File: app/views/errors/500.haml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  #jt-error label { display: block; }
+  #jt-error input, textarea { font-size: 16px; margin-bottom: 16px; width: 300px; display: block; }
+
+#jt-error
+  %h1 Server Error
+
+  %p Sorry!  Something went wrong with our server. Our development team has been notified.
+  %p Error Code: 500 (#{@exception.class.name})
+
+  / = form_tag "/jefferies_tube_add_error_information", method: "post" do
```
Comment (Nonsense331) [inline app/views/errors/500.haml]: It would be awesome if this worked

**Comment by Nonsense331** [comment]
🚢

File: app/views/errors/500.haml (hunk 1 above)
Comment (samsonasu) [inline app/views/errors/500.haml]: yeah, I put in a support ticket with rollbar to help

**Comment by petitJAM** [comment]
delete your branches maaan

**Pull request merged**